lib/cli.ts

```diff
@@ -9,6 +9,17 @@ import ConsoleUI from '../types/console-ui';
 import WatchDetector from 'watch-detector';
 import UI from 'console-ui';
 
+enum EnvironmentType {
```

Comment:
I'm not sure if broccoli imposes any specific environment name, rather it's for the consumer to decide.
This may be better off as a string

Reply:
It's explicitly defined in cli options

![image](https://user-images.githubusercontent.com/1360552/134963884-ef1d9dd1-0488-4c5d-8ae4-33937c97be45.png)

Reply:
of course environments could be different (user-defined), but inside broccoli codebase we support only 2 options (explicitly), and we don't expose this type as part of public API